project/driver_controls/inc/input_event.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+// Define common typedefs for the various FSMs to use.
+
+#include "event_queue.h"
+
+// Typedef for the FSM arbitration functions
+typedef bool (*InputEventCheck)(Event *e);
```

Comment:
It's still a little weird to have it here when it's really from your other module, you know?